CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Added
+
+- Add priority arbiter
+- Add Pseudo Least Recently Used tree
+
+### Changed
+- Add priority arbiter implementation
```

Comment:
Why this entry when there is already a "Add priority arbiter" entry? And why is it under _Changed_?